lms/djangoapps/discussion_api/serializers.py

```diff
@@ -187,6 +187,7 @@ class ThreadSerializer(_ContentSerializer):
     non_endorsed_comment_list_url = serializers.SerializerMethodField("get_non_endorsed_comment_list_url")
     read = serializers.BooleanField(read_only=True)
     has_endorsed = serializers.BooleanField(read_only=True, source="endorsed")
+    response_count = serializers.IntegerField(source="resp_total", read_only=True)
```

Comment:
Whoops, poorly worded question. To clarify, is there actually a `resp_total` field that is returned from the ccs?